params/protocol_params.go

```diff
@@ -79,7 +79,8 @@ const (
 	MemoryGas        uint64 = 3     // Times the address of the (highest referenced byte in memory + 1). NOTE: referencing happens on read, write and in instructions such as RETURN and CALL.
 	TxDataNonZeroGas uint64 = 68    // Per byte of data attached to a transaction that is not equal to zero. NOTE: Not payable on data of calls between transactions.
 
-	MaxCodeSize = 24576 // Maximum bytecode to permit for a contract
+	OriginalMaxCodeSize = 24576 // Maximum bytecode to permit for a contract
+	MaxCodeSize         = 32768 // Maximum bytecode to permit for a contract
```

Comment:
to avoid clashing with upstream, i suggest to keep the `MaxCodeSize` and have our own `QuorumMaxCodeSize`

Reply:
Agree. changed accordingly